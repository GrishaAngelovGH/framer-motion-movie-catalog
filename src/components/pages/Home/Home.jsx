import Layout from "components/Layout"
import Header from "components/Header"

const Home = () => (
  <Layout header={<Header />}>
    <div className="bg-blue-300 h-full">Content</div>
  </Layout>
)

export default Home