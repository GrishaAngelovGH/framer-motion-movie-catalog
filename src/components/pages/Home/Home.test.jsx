import { render } from "@testing-library/react"

import Home from "./Home"

test("should render pages/Home component", () => {
  const view = render(<Home />)

  expect(view).toMatchSnapshot()
})