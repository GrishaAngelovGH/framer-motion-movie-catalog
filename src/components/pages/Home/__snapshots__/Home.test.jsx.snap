// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render pages/Home component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col h-screen"
      >
        <div
          class="h-[60px]"
        >
          <div
            class="flex justify-between items-center bg-blue-500 h-full"
          >
            <div
              class="ms-5 w-[200px] flex justify-between items-center"
              style="opacity: 0; transform: scale(0.5) translateZ(0);"
            >
              <img
                class="w-[40px]"
                src="/images/film-roll.png"
              />
              <h1
                class="text-4xl font-bold text-white"
              >
                Cinema
              </h1>
            </div>
            <div
              class="me-5"
            >
              <a
                class="text-white hover:text-white hover:underline"
                href="/"
              >
                Home
              </a>
            </div>
          </div>
        </div>
        <div
          class="h-full"
        >
          <div
            class="bg-blue-800 min-h-full flex flex-col md:flex-row justify-center items-center"
          >
            <div
              class="flex flex-col"
              style="transform: translateX(-100%) translateZ(0);"
            >
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/the-garfield-movie.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      The Garfield Movie
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/fly-me-to-the-moon.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Fly Me to the Moon
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/kung-fu-panda-4.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Kung Fu Panda 4
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/kingdom-of-the-planet-of-the-apes.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Kingdom of the Planet of the Apes
                    </p>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="flex flex-col"
              style="transform: translateX(-200%) translateZ(0);"
            >
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/transformers-one.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Transformers One
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/mufasa-the-lion-king.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Mufasa: The Lion King
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/beverly-hills-cop-axel-f.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Beverly Hills Cop: Axel F
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/indiana-jones-and-the-dial-of-destiny.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Indiana Jones and the Dial of Destiny
                    </p>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="flex flex-col"
              style="transform: translateX(-300%) translateZ(0);"
            >
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/gran-turismo.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Gran Turismo
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/aquaman-and-the-lost-kingdom.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Aquaman and the Lost Kingdom
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/the-super-mario-bros-movie.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      The Super Mario Bros. Movie
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/spider-man-across-the-spider-verse.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Spider-Man: Across the Spider-Verse
                    </p>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="flex flex-col"
              style="transform: translateX(-400%) translateZ(0);"
            >
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/shooting-stars.png"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Shooting Stars
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/sweetwater.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      Sweetwater
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/a-million-miles-away.png"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      A Million Miles Away
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="flex"
              >
                <div
                  class="text-center"
                >
                  <div
                    class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                  >
                    <img
                      class="rounded-lg min-h-[350px]"
                      src="images/the-marvels.jpg"
                    />
                    <p
                      class="text-xl text-white h-[90px] mt-2"
                      style="opacity: 0;"
                    >
                      The Marvels
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col h-screen"
    >
      <div
        class="h-[60px]"
      >
        <div
          class="flex justify-between items-center bg-blue-500 h-full"
        >
          <div
            class="ms-5 w-[200px] flex justify-between items-center"
            style="opacity: 0; transform: scale(0.5) translateZ(0);"
          >
            <img
              class="w-[40px]"
              src="/images/film-roll.png"
            />
            <h1
              class="text-4xl font-bold text-white"
            >
              Cinema
            </h1>
          </div>
          <div
            class="me-5"
          >
            <a
              class="text-white hover:text-white hover:underline"
              href="/"
            >
              Home
            </a>
          </div>
        </div>
      </div>
      <div
        class="h-full"
      >
        <div
          class="bg-blue-800 min-h-full flex flex-col md:flex-row justify-center items-center"
        >
          <div
            class="flex flex-col"
            style="transform: translateX(-100%) translateZ(0);"
          >
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/the-garfield-movie.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    The Garfield Movie
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/fly-me-to-the-moon.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Fly Me to the Moon
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/kung-fu-panda-4.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Kung Fu Panda 4
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/kingdom-of-the-planet-of-the-apes.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Kingdom of the Planet of the Apes
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col"
            style="transform: translateX(-200%) translateZ(0);"
          >
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/transformers-one.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Transformers One
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/mufasa-the-lion-king.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Mufasa: The Lion King
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/beverly-hills-cop-axel-f.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Beverly Hills Cop: Axel F
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/indiana-jones-and-the-dial-of-destiny.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Indiana Jones and the Dial of Destiny
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col"
            style="transform: translateX(-300%) translateZ(0);"
          >
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/gran-turismo.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Gran Turismo
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/aquaman-and-the-lost-kingdom.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Aquaman and the Lost Kingdom
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/the-super-mario-bros-movie.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    The Super Mario Bros. Movie
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/spider-man-across-the-spider-verse.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Spider-Man: Across the Spider-Verse
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col"
            style="transform: translateX(-400%) translateZ(0);"
          >
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/shooting-stars.png"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Shooting Stars
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/sweetwater.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    Sweetwater
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/a-million-miles-away.png"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    A Million Miles Away
                  </p>
                </div>
              </div>
            </div>
            <div
              class="flex"
            >
              <div
                class="text-center"
              >
                <div
                  class="m-8 mb-0 md:w-[160px] lg:w-[220px]"
                >
                  <img
                    class="rounded-lg min-h-[350px]"
                    src="images/the-marvels.jpg"
                  />
                  <p
                    class="text-xl text-white h-[90px] mt-2"
                    style="opacity: 0;"
                  >
                    The Marvels
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
